model/patch/patch.go

```diff
@@ -195,6 +198,18 @@ func (p *Patch) SetDescription(desc string) error {
 	)
 }
 
+func (p *Patch) SetEnqueued(newPatchID string) error {
```

Comment:
can this be SetMergePatch instead of SetEnqueued, and/or put a comment here on when to use this function? Since we use queues for so many things I think this description can be misleading.

Reply:
I renamed it to `SetMergePatch`